package.json

```diff
@@ -32,37 +31,36 @@
     "type": "git",
     "url": "https://github.com/danrevah/ngx-pipes.git"
   },
+  "peerDependencies": {
+    "@angular/core": "^4.0.1 || ~5.0.0"
+  },
   "devDependencies": {
-    "@angular/common": "^4.0.1",
-    "@angular/compiler": "^4.0.1",
-    "@angular/core": "^4.0.1",
-    "@angular/forms": "^4.0.1",
-    "@angular/http": "^4.0.1",
-    "@angular/platform-browser": "^4.0.1",
-    "@angular/platform-browser-dynamic": "^4.0.1",
-    "@angular/router": "^4.0.1",
-    "zone.js": "^0.8.4",
-    "coveralls": "^2.11.15",
-    "gulp": "^3.9.1",
-    "gulp-clean": "^0.3.2",
-    "rollup": "^0.41.4",
-    "rollup-plugin-node-resolve": "^2.0.0",
-    "rollup-plugin-uglify": "^1.0.1",
-    "@angular/cli": "1.0.0",
-    "@angular/compiler-cli": "^4.0.1",
-    "@types/jasmine": "2.5.38",
-    "@types/node": "~6.0.60",
-    "jasmine-core": "~2.5.2",
-    "jasmine-spec-reporter": "~3.2.0",
-    "karma": "~1.4.1",
-    "karma-chrome-launcher": "~2.0.0",
+    "@angular/cli": "^1.5.0",
+    "@angular/common": "~5.0.0",
+    "@angular/compiler": "~5.0.0",
+    "@angular/compiler-cli": "~5.0.0",
+    "@angular/core": "~5.0.0",
+    "@angular/platform-browser": "~5.0.0",
+    "@angular/platform-browser-dynamic": "~5.0.0",
+    "@types/jasmine": "~2.6.0",
+    "@types/jasminewd2": "~2.0.2",
+    "@types/node": "~8.0.0",
+    "codelyzer": "^4.0.1",
+    "core-js": "^2.4.1",
+    "jasmine-core": "~2.8.0",
+    "jasmine-spec-reporter": "~4.2.0",
+    "karma": "~1.7.0",
+    "karma-chrome-launcher": "^2.2.0",
     "karma-cli": "~1.0.1",
-    "karma-coverage-istanbul-reporter": "^0.2.0",
+    "karma-coverage-istanbul-reporter": "^1.3.0",
     "karma-jasmine": "~1.1.0",
     "karma-jasmine-html-reporter": "^0.2.2",
-    "protractor": "~5.1.0",
-    "ts-node": "~2.0.0",
-    "tslint": "~4.5.0",
-    "typescript": "~2.2.0"
+    "ng-packagr": "^1.5.0",
+    "protractor": "^5.2.0",
+    "standard-version": "^4.2.0",
+    "ts-node": "^3.0.0",
+    "tslint": "^5.5.0",
+    "typescript": "~2.5.0",
```

Comment:
@danrevah I'm sorry, i forgot to fix this version. we'd better not upgrade the typescript bigger than 2.4. Because Angular doesn't support it now. I've tried to upgrade to v2.5 in another package, with rollup, it produces incorrect output. So, upgrading typescript is not proposed.

Reply:
you have to downgrade it, like `typescript: ~2.4.0`

Reply:
It's 2.4.0 in the merge, and this commit has been merged and released in version v2.0.0